### ISSUE TITLE
Cargo Ship not opening on Big Sur

### ISSUE THREAD

**Issue by BurroHen** [univ-of-utah-marriott-library-apple/scl_jamf_tools#1]
I previously ran Cargo Ship under Catalina.   Fired it up today and it refused to open.  Running via command line generated:

INFO: Running Cargo Ship
INFO: Level: Method/function: Message
INFO: read_create_prefs: activated
invalid literal for int() with base 10: 's'
INFO: Configuration path: /Users/burrohen/Library/Preferences/edu.scl.utah.cargo_ship.ini
INFO: injest_prefs: activated
INFO: Configuration file variables:
INFO: login
INFO: 	username ''
INFO: 	hosts 'REDACTED'
macOS 11 or later required !
zsh: abort      ./Cargo\ Ship.app/Contents/MacOS/Cargo\ Ship

**Comment by uurazzle** [comment]
Hi @BurroHen:

Sorry about the issue. So from the output the system you are running Cargoship on is macOS 11.x?

We are busy with other projects and priorities, but will add it to our todos and troubleshoot the issues as soon as we can and let you know status.

**Comment by BurroHen** [comment]
That is correct.  I am currently running macOS 11.1 x86.   No worries, just wanted to give you a heads up.  Thank you for the tool btw.

**Comment by uurazzle** [comment]
Hi @BurroHen:

ok, thx for the information, we will try to get to it when we have bandwidth and are able. If you know any python pull requests are recommended ;-)
